office/libreoffice/Portfile

```diff
@@ -208,6 +210,7 @@ configure.args-append  \
     --enable-readonly-installset \
     --enable-release-build \
     --enable-symbols \
+    --with-boost=[boost::install_area] \
```

Comment:
I believe this is also done by the portgroup?

Reply:
hmmm ... In my experience the primary issue with non-CMake configuration is that the flags can/will vary even between different ports. Broadly speaking `--with-boost=...` will work with autotools, but it's not guaranteed to do so. With CMake there are standard ways for finding, and so what was added is virtually universally correct across all ports -- I know of no exceptions, but that doesn't mean there aren't any!